File: Coding/Java/Set3/SplitNumberToDigits.java
package Set3;


import java.util.ArrayList;

public class SplitNumberToDigits {
    public static ArrayList<Integer> splitNumber(int num){
        ArrayList<Integer> digits  = new ArrayList<>();
        while(num != 0) {
            int digit = num % 10;
            digits.addFirst(digit);
            num /= 10;
        }
        return digits;
    }
    public static void main(String[] args) {
        System.out.println(splitNumber(100));
    }
}

File: Coding/Python/Set3/SplitNumberToDigits.py
def split_number_digits(num):
    result = []
    while num != 0:
        digit = num % 10
        result.insert(0, digit)
        num //= 10
    return result


print(split_number_digits(100))
print(split_number_digits(101))
print(split_number_digits(1))

